internal/engine/interpreter/interpreter.go

```diff
@@ -300,8 +300,19 @@ func (e *engine) lowerIR(ir *wazeroir.CompilationResult) (*code, error) {
 			op.sourcePC = ir.IROperationSourceOffsetsInWasmBinary[i]
 		}
 		switch o := original.(type) {
-		case wazeroir.OperationBuiltinFunctionCheckExitCode:
-		case wazeroir.OperationUnreachable:
+		case wazeroir.OperationNullary:
```

Comment:
do you want to proceed how this can be done in the union struct? In the end, you have to delete `wazeroir.OperationNullary` and replace it with the union struct, no?

Reply:
sure that is correct. The only reason I introduced `OperationNullary` is only to make this PR fully self-contained

Reply:
no need to make it self-contained, but I appreciate what you intended to do! let's just make a shortcut to the goal of nuking the old operation